test/benchmark/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 set(TEST_TYPE "BENCHMARK")
 
-include(IgnBenchmark OPTIONAL RESULT_VARIABLE IgnBenchmark_FOUND)
+include(GzBenchmark OPTIONAL RESULT_VARIABLE GzBenchmark_FOUND)
```

Comment:
I think this will need a change in `gz-cmake`:

https://github.com/gazebosim/gz-cmake/blob/main/cmake/IgnBenchmark.cmake

Reply:
I'll ticktock it :thinking: 
https://github.com/gazebosim/gz-sim/compare/f864ede73d1c3bc590e38c99fd4f890b91a932fe..be1767359cc93b7736c94a3a34344b71f12c447a

https://github.com/gazebosim/gz-sim/compare/be1767359cc93b7736c94a3a34344b71f12c447a..f6166764064941f7f713488ca9bf6041ab95d7f0